### ISSUE TITLE
D.C.

### ISSUE THREAD

**Issue by The2Earthlings** [kwindrem/GuiMods#286]

**Comment by kwindrem** [comment]
This is true. Gui V1 (on which GuiMods is based) did not show any secondary shunt information. 

In order for GuiMods to show a device int the other DC devices section it must provide it's own service which generally means the decvice would need it's own shunt.

I could make an exception and show secondary shunt voltages in the list but since GuiMods is going ways soon I'm not inclined to make changes now.

**Comment by The2Earthlings** [comment]
Thank you for the info.
